specs-go/config.go

```diff
@@ -469,6 +462,23 @@ type WindowsNetworkResources struct {
 	EgressBandwidth *uint64 `json:"egressBandwidth,omitempty"`
 }
 
+// LinuxSeccomp represents syscall restrictions
+type LinuxSeccomp struct {
+	DefaultAction LinuxSeccompAction `json:"defaultAction"`
+	// Architectures is kept to maintain backward compatibility for projects
+	// that already use the old seccomp profile.
+	Architectures []Arch          `json:"architectures,omitempty"`
+	ArchMap       []Architecture  `json:"archMap,omitempty"`
```

Comment:
> Architectures2 doesn't really give it any meaning...

Like `dup2` (and 3 and 4) the meaning is "This is what we should have done for `architectures{n-1}`, but we aren't changing that because we're preserving backwards compatible".  But if `architectures` was poorly named to begin with (I don't know), picking a better name now makes sense.